CHANGELOG.rst

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+1.2.2 (2022-01-19)
+------------------
+
+- [NEW] Added Kazakh locale.
+- [FIX] The Czech, Slavic and Slovak locales now support ``dehumanize``.
```

Comment:
For Slavic locales, do we want to be more specific?  I.E Russian, Polish, etc. ?

Reply:
Probably a good idea.